tests/ci_build/Dockerfile.cpu

```diff
@@ -5,7 +5,11 @@ ENV DEBIAN_FRONTEND noninteractive
 SHELL ["/bin/bash", "-c"]   # Use Bash as shell
 
 # Install all basic requirements
+ENV http_proxy=http://child-prc.intel.com:913
+ENV https_proxy=http://child-prc.intel.com:913
 RUN \
+    echo 'Acquire::http::proxy "http://child-prc.intel.com:913";' >> /etc/apt/apt.conf && \
+    echo 'Acquire::https::proxy "http://child-prc.intel.com:913";' >> /etc/apt/apt.conf && \
```

Comment:
Why do we need to use a proxy at all? Can we install Arrow from PyPI or Conda?

Reply:
I should've remote these lines before commit. Will fix it. Thanks!